test/mux.sh

```diff
@@ -237,15 +250,25 @@ write_and_verify_logs() {
 }
 
 reset_ES_HOST() {
-    oc set env dc logging-mux $1 $2
+    muxpod=$( get_running_pod mux )
+    os::cmd::try_until_success "oc set env dc logging-mux $1 $2"
```

Comment:
I think you can just use `expect_success` here instead of `try_until_success` - that is, `oc set env` should be run once, and only once.